plugins/CoreHome/vue/src/EnrichedHeadline/EnrichedHeadline.vue

```diff
@@ -56,6 +56,8 @@
       v-show="showInlineHelp"
     >
       <div v-html="$sanitize(actualInlineHelp)"/>
+      <span class="helpDate"
+            v-html="reportGenerated"></span>
```

Comment:
Should this be `$sanitize(reportGenerated)`? v-html doesn't perform any sanitization.

Reply:
The `reportGenerated` should contain text only, which is coming from a translation. Guess it would not hurt to add it.